### ISSUE TITLE
When watching the video, it will enter the lock screen

### ISSUE THREAD

**Issue by clutch-man** [phaselden/FlipIt#23]
Don’t want to enter the lock screen while watching a video.
Hope to add a judgment logic.

**Comment by SmilerRyan** [comment]
I don't believe this is a problem with this screensaver perticuarly, but with the way screensavers work in general.
As mentioned by many others online, you'll have to turn your screensaver off or have your mouse move (maybe a [Mouse Jiggler](https://github.com/cerebrate/mousejiggler/releases) app helps you).

**Comment by clutch-man** [comment]
I don't know if there is something wrong with the update of Edge browser. 
Thank you for your prompt reply anyway

**Comment by nshopik** [comment]
Screensaver doesn't decided when its turn on, windows os is. Most fullscreen apps usually block screensaver activation. FlipIt doesn't control that behavior only when turn off if you move mouse.

**Comment by clutch-man** [comment]
> Screensaver doesn't decided when its turn on, windows os is. Most fullscreen apps usually block screensaver activation. FlipIt doesn't control that behavior only when turn off if you move mouse.
yea
In fact, when I watch local videos, the screensaver won't appear after the scheduled time.
But when I watch a video online in my browser, the screensaver appears.
A recent update to the Edge browser may have caused this situation, but it didn't happen before.

**Comment by clutch-man** [comment]
> Screensaver doesn't decided when its turn on, windows os is. Most fullscreen apps usually block screensaver activation. FlipIt doesn't control that behavior only when turn off if you move mouse.

yea
In fact, when I watch local videos, the screensaver won't appear after the scheduled time.
But when I watch a video online in my browser, the screensaver appears.
A recent update to the Edge browser may have caused this situation, but it didn't happen before.
